tests/test_read_algos.test

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env bash
+# Copyright (C) The libssh2 project and its contributors.
+# SPDX-License-Identifier: BSD-3-Clause
 
 # Copyright (C) Viktor Szakats
```

Comment:
This line could replace the generic Copyright header in line 2.

Reply:
:+1: